reVX/rpm/rpm_clusters.py

```diff
@@ -322,13 +323,17 @@ def _get_cluster_geom(gdf_points):
         return clusters, mean_dist
 
     @classmethod
-    def _generate_shapefile(cls, meta, fpath, beautify=True):
+    def generate_shapefile(cls, meta, fpath, beautify=True,
+                           source_crs="EPSG:4326", target_crs=None):
         """
         Generate cluster polygons and save to shapefile
         """
         geometry = [Point(xy) for xy in zip(meta.longitude, meta.latitude)]
         gdf_points = gpd.GeoDataFrame(meta, geometry=geometry,
-                                      crs="EPSG:5070")
+                                      crs=source_crs)
+
+        if target_crs is not None:
```

Comment:
Do you know if Geopandas efficiently handles the case where the df crs == target crs? If we want to be extra cautious, we could change this to `if target_crs != source_crs:` in order to be completely sure that no unnecessary computation is happening

Reply:
No idea haha that's a good idea to add that!